index.html

```diff
@@ -229,38 +229,71 @@ <h3>
             <dt id="data-integrity-1" class="spec">Verifiable Credential Data Integrity 1.0</dt>
             <dd>
               <p>
-This specification defines how to express proofs of integrity, such as digital
-signatures or proofs of existence, for bounded documents, such as verifiable
-credentials. Concrete serializations will be provided based on <a
-href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>, the <a
-href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>, 
-the <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1
-Cryptosuite</a>, and for <a href="https://github.com/w3c-ccg/lds-jws2020/">
-JSON Web Signature 2020</a>. Concrete serializations based on the <a
-href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a> might be provided, if an <a
-href="https://identity.foundation/bbs-signature/">IETF BBS+ RFC</a> is published
-in time. Concrete serializations for VC-JWP might be provided, if an <a
-href="https://json-web-proofs.github.io/json-web-proofs/">IETF JWP RFC</a> is
-published in time. 
-The final output specifications might not map directly to
-the list of input drafts. Other output serializations covering cryptographic
-primitives published in IETF RFCs not mentioned in this paragraph might be
-produced.
+This family of specifications defines how to express proofs of integrity for
+verifiable credentials using a number of concrete serializations for each of the
+defined syntaxes. The specific set of concrete serializations included will be
+determined by the Working Group. The following are a non-exhaustive selection
+of expected input documents:
               </p>
-
-              <p class="draft-status"><b>Draft state:</b>
-                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>,
+              <p class="input-documents"><b>Container Formats:</b>
                 <a href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>,
-                <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>,
-                <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1 Cryptosuite</a>,
-                <a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a>,
-                <a href="https://github.com/w3c-ccg/lds-jws2020/">JSON Web Signature 2020</a>              
+                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>
```

Comment:
> Ahh I see, in that case, I retract the change request... 

Great, thank you!

> but still have concerns about the crypto suites as the relate to 1, or 2.

Yes, and I expect the concern there are "the WG may not do all of these, or they may have some that need to be done that we don't know about right now, or we may choose to structure everything differently." 

I think @brentzundel's language that he added (and I agree with): "The specific set of concrete serializations included will be determined by the Working Group. The following are a non-exhaustive selection of expected input documents:" addresses some of that concern.

However, I do agree with you that we might need to further refine the two categories if we get the group to agree to structure the charter in this new way. I expect more PRs to clarify those things /if/ we get this in place. I'm also hoping @mprorock's PR #66 makes it in since it only has one objector.

> but I think the restructuring is so desperately needs I will hold on any further comments, until we have it.

+1, thanks... I'm attempting to do what you've been asking for here for a while, but instead of fast merges, I'm trying to take all of those "fast merges" into this PR, so we can just get this PR and then close all the other PRs... effectively accomplishing the same thing.

Reply:
data integrity CCG draft (former LDP) is not part of 2?

Reply:
> data integrity CCG draft (former LDP) is not part of 2?

No, it isn't, because the incubation is done and it's ready for a WG. That doesn't mean that the input documents won't change wildly from what they are currently. We are just signalling that one of the major reasons this WG is being formed is to standardize Data Integrity (and cryptosuites), and that is one of the primary purposes of our charter, and if we don't do that, we will have failed as a WG. Everything under "normative deliverables" fits into that category -- the expectation is that the WG will be delivering some variation of those documents as a REC by the end of the WG.